nixos/tests/tor.nix

```diff
@@ -1,8 +1,8 @@
-import ./make-test-python.nix ({ lib, ... }: with lib;
+import ./make-test-python.nix ({ lib, ... }:
```

Comment:
The placement of this with lib is unexpected. If it wasn't removed I would at least have moved it to a more normal position.